docs/src/main/sphinx/language/types.rst

```diff
@@ -262,11 +262,26 @@ The following examples illustrate the behavior::
 Instant in time that includes the date and time of day with ``P`` digits of
 precision for the fraction of seconds and with a time zone. Values of this
 type are rendered using the time zone from the value.
-Time zones are expressed as the numeric UTC offset value::
+Time zones can be expressed in the following ways:
+
+* ``UTC``, with ``GMT``, ``Z``, or ``UT`` usable as aliases for UTC.
+* ``+hh:mm`` or ``-hh:mm`` with ``hh:mm``, with or without ``UTC`` or an alias,
```

Comment:
> an alias,

did you actually check which aliases work?

Reply:
UTC, GMT, and UT work as aliases with this syntax. Not Z.

Reply:
Should we have tests for this in the codebase @findepi ?